include/exadg/time_integration/time_int_multistep_base.h

```diff
@@ -176,20 +169,20 @@ class TimeIntBDFBase : public TimeIntBase
    * Initializes the solution vectors by prescribing initial conditions or reading data from
    * restart files and initializes the time step size.
    */
-  virtual void
-  initialize_solution_and_time_step_size(bool do_restart);
+  void
+  initialize_vectors_and_time_step_size(bool do_restart);
 
   /*
-   * Initializes the solution vectors at time t
+   * Initializes vectors needed for the multistep scheme
    */
   virtual void
-  initialize_current_solution() = 0;
+  initialize_multistep_dof_vectors() = 0;
 
   /*
-   * Initializes the solution vectors at time t - dt[1], t - dt[1] - dt[2], etc.
+   * Initializes the initial solution
    */
   virtual void
-  initialize_former_solutions() = 0;
+  initialize_current_solution() = 0;
```

Comment:
Please undo the change regarding switching the order of function definitions in this PR.